From c52ca58a7a8b9f87bde7ca1ab5b11a39d82854e5 Mon Sep 17 00:00:00 2001
Message-Id: <c52ca58a7a8b9f87bde7ca1ab5b11a39d82854e5.1411757522.git.jen@redhat.com>
From: Jeff Nelson <jenelson@redhat.com>
Date: Fri, 26 Sep 2014 16:39:43 -0400
Subject: [CHANGE 1/7] Revert "migration: Don't calculate bandwidth when last
 cycle was too short"
To: rhvirt-patches@redhat.com,
    jen@redhat.com

RH-Author: Jeff Nelson <jenelson@redhat.com>
Message-id: <d463a80cccde4cdbfc914438ecc6de73bfa74d15.1411747689.git.jen@redhat.com>
Patchwork-id: 61458
O-Subject: [RHEL-6.6.z qemu-kvm PATCH 1/7] Revert "migration: Don't calculate bandwidth when last cycle was too short"
Bugzilla: 970103
RH-Acked-by: Dr. David Alan Gilbert (git) <dgilbert@redhat.com>

This reverts commit 20851e2f5852308d417ddcf8a1790f4bb0cdcb93.

Signed-off-by: Jeff E. Nelson <jen@redhat.com>
---
 vl.c | 14 --------------
 1 file changed, 14 deletions(-)

Signed-off-by: Jeff E. Nelson <jen@redhat.com>
---
 vl.c | 14 --------------
 1 file changed, 14 deletions(-)

diff --git a/vl.c b/vl.c
index 3b37ccd..a078440 100644
--- a/vl.c
+++ b/vl.c
@@ -2948,20 +2948,6 @@ static int ram_save_live(Monitor *mon, QEMUFile *f, int stage, void *opaque)
         uint64_t expected_time;
         double bwidth = 0;
 
-        /*
-         * If it is smaller than 10ms, we don't trust the measurement.
-         *
-         * Sometimes as the amount of time is very small and that
-         * makes the bandwidth really, really high.  But if
-         * networking is very flaky, we also want to finish migration
-         * when the amount of memory pending is small enough. (10MB
-         * on this case).
-         */
-
-        if ((ram_bytes_remaining() > 10 * 1024 * 1024) && (t0 < 10000000)) {
-            return 0;
-        }
-
         bwidth = ((double)bytes_transferred - bytes_transferred_last) / t0;
 
         /*
-- 
1.9.3

